src/comp/comp.py

```diff
@@ -24,48 +24,50 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [i.name for i in humans if (i.name[0] == 'D')]
```

Comment:
for future reference, you actually don't need the parentheses wrapping the if portion of a comprehension.
a = [i.name for i in humans if i.name[0] == 'D'] works just the same. and that goes the same for all the answers below as well.